src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+[[“One beer. Extra piss.”|The Gutter - Ron - Weak drink]]
+
+::The Gutter - Ron - Won too soon
+Ron stared at me dumbfounded. I got a little too jazzed and slapped the pool stick out of his hand while shouting “whose house?!” I did a chicken dance that I’m not proud of, but it felt amazing in the moment.
+
+Ron sulked up to the bar to grab my drink. I shouted after him, “don’t bother!”
+
+[[“Let’s get another round in. Same bet.”|The Gutter - Ron - Pool game start]]
+
+::The Gutter - Ron - Strong drink
+Toe dumped half a beer, filled the can with whiskey, and handed it to me. I delivered the mess to Ron and he took it down in one victory gulp.
+
+“One more round!” he screamed in my face like a hell dad. “And this time, I’ll give you that fucking letter if you win. But if I win, I get to fuck your mom! Oh wait; I can do that whenever. Hahahahaha.”
+
+[[“Let’s play, you shitlord.”|The Gutter - Ron - Pool victory]]
+
+::The Gutter - Ron - Weak drink
+I passed Ron his beer and he chugged it with abandon. It wasn’t fun to watch and then, when he said “let’s go again,” that wasn’t fun to hear. But I knew there had to be a way to trick his dumb ass if I kept at it.
+
+[[“Fine. I’ll do another round.”|The Gutter - Ron - Pool game start]]
+
+::The Gutter - Ron - Pool victory
+Ron was so shitfaced that he couldn’t hold the cue right. I took my time, letting him get even more drunk and exhausted. When I sank the eight ball, Ron collapsed on the table.
+
+I poked him hard in the tit with a pool cue to prop his body up, and grabbed the letter from his pocket. I noticed his keys jangling on his side too.
+
+(link: "I took Ron’s keys, even the one to his shitty golf cart, and headed for the exit")[(set: $cartKeyTaken to true)(go-to: "The Gutter - Ending")].
+(link: "I took the key to the Church door, but left the key to Ron’s shitty golf cart. I made a beeline for the exit")[(set: $cartKeyTaken to false)(go-to: "The Gutter - Ending")].
+
+::The Gutter - Pool table - First approach
+The teens didn’t move an inch. In fact, they seemed to be rounding first base.
+
+[I coughed.]<poolTableCough|
+[[[I turned back and let the kids have their fun|The Gutter - Inside hub]].]<poolTableExit1|
+\(click: ?poolTableCough)[(replace: ?poolTableExit1)[$dissolve[They were still ignoring me. Clothes were coming off and the only benefit was that I got to see a tattoo of a hatchet man dressed as Jesus pissing on Satan.
+
+[I kicked the leg of the goddamn table.]<poolTableKick|
+[[[I turned back. They were getting pretty close to just fucking at this point|The Gutter - Inside hub]].]<poolTableExit2|]]]
+\(click: ?poolTableKick)[(replace: ?poolTableExit2)[$dissolve["What's your fucking problem, sis?" one of the teens said, in-between tongue thrusts. "Go ruin somebody else's shit.”]]]
+
+(click: "ruin")[$dissolve[(set: $poolTableAttemptMade to true)[I frowned in a way that made me feel very old. [[Guess these horny ass kids weren’t moving for me|The Gutter - Inside hub]].]]]
```

Comment:
"horny-ass" is a compound adjective, so it should be hyphenated.

This is probably one of the weirdest comments left in GitHub history.

Reply:
Don't worry, I just searched GitHub for "horny" and there's tons of awful stuff on here. For instance, "Like Wario Land 3, but hornier" is something I found.